modules/service_accounts_iam/main.tf

```diff
@@ -30,7 +30,7 @@ module "helper" {
  *****************************************/
 resource "google_service_account_iam_binding" "service_account_iam_authoritative" {
   for_each           = module.helper.set_authoritative
-  service_account_id = "projects/${var.project}/serviceAccounts/${module.helper.bindings_authoritative[each.key].name}"
+  service_account_id = "projects/${var.project}/serviceAccounts/${module.helper.bindings_authoritative[each.key].name}@${var.project}.iam.gserviceaccount.com"
```

Comment:
Looking at our tests we have been using emails rather than SA names (as show in [our example in Readme](https://github.com/terraform-google-modules/terraform-google-iam/tree/master/modules/service_accounts_iam#example-usage))
https://github.com/terraform-google-modules/terraform-google-iam/blob/91929548c9a521de033844fddc6391d4237a5dfc/test/fixtures/helper/base/outputs.tf#L30-L33
https://github.com/terraform-google-modules/terraform-google-iam/blob/91929548c9a521de033844fddc6391d4237a5dfc/test/fixtures/helper/iam.tf#L67-L72

Users also seem to be using email
https://github.com/terraform-google-modules/terraform-google-iam/issues/105
https://github.com/search?l=HCL&q=service_accounts_iam&type=Code

API doc https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts confirms that it should have been email and I cant seem to find any info that this is a recent change.

Should we just update our example to use email and prevent a breaking change?

Reply:
Make sense. Leme revert the change and update README with proper example